api/sdmx-rest.yaml

```diff
@@ -239,6 +239,57 @@ paths:
         <<: *common_responses
         '200':
           $ref: '#/components/responses/200-meta'
+
+  /registration/id/{registrationID}:
+    get:
+      summary: 'Registration queries (by registration ID)'
+      tags: 
+        - Registration queries
+      description: |
+        These queries enable clients to request a data registration by it's unique identifier
```

Comment:
Should it be `its` instead of `it's`?

Reply:
Wow, you're right - I thought it's was possesive, but after checking I have found out that was in common use about 300 years ago and now its is used, I'm really showing my age ;-)

Reply:
Thanks, I learned something :)